srcpkgs/intellij-idea-community-edition/template

```diff
@@ -48,7 +47,6 @@ do_install() {
 		rm ${DESTDIR}/usr/lib/intellij-idea/bin/libdbm64.so
 		rm ${DESTDIR}/usr/lib/intellij-idea/lib/pty4j-native/linux/x86_64 -rf
 		rm ${DESTDIR}/usr/lib/intellij-idea/lib/pty4j-native/linux/ppc64le -rf
-		rm ${DESTDIR}/usr/lib/intellij-idea/plugins/android/lib/libwebp/linux/libwebp_jni64.so
```

Comment:
Why to preserve x86_64 binary in i686, and the other way too?

Reply:
Sorry, I thought they had deleted it but, just now I just looked at it and I saw that they have simply changed its location

Reply:
The new location is: ${DESTDIR}/usr/lib/intellij-idea/plugins/**webp**/lib/libwebp/linux/libwebp_jni.so

Reply:
@Chocimier when i try to git clone i get this error:
`Invalid path 'common/repo-keys/3d:b9:c0:50:41:a7:68:4c:2e:2c:a9:a2:5a:04:b7:3f.plist' unable to checkout working tree warning: Clone succeeded, but checkout failed. You can inspect what was checked out with 'git status' and retry with 'git restore --source=HEAD :/'`
Can I redo the pull request?

Reply:
Best if you sort out (maybe remove and recreate if needed) local copy, then force push to your remote branch.